src/ripple/overlay/impl/TrafficCount.h

```diff
@@ -223,6 +229,51 @@ class TrafficCount
         { "getobject (get)" },                                    // category::get_hash
         { "unknown" }                                             // category::unknown
     }};
+
+    std::array<std::string, category::unknown+1> monNames_
```

Comment:
@nbougalis I had to make a new array with StatsD-friendly names corresponding to each original category name. I tried my best to keep them similar to the original names. Let me know if you had something else in mind.